spec/calendar.html

```diff
@@ -294,6 +294,9 @@ <h1>ToTemporalCalendar ( _temporalCalendarLike_ )</h1>
           1. If ? HasProperty(_temporalCalendarLike_, *"calendar"*) is *false*, return _temporalCalendarLike_.
           1. Set _temporalCalendarLike_ to ? Get(_temporalCalendarLike_, *"calendar"*).
           1. If Type(_temporalCalendarLike_) is Object and ? HasProperty(_temporalCalendarLike_, *"calendar"*) is *false*, return _temporalCalendarLike_.
+          1. Let _identifier_ be ? ToString(_temporalCalendarLike_).
+          1. If IsBuiltinCalendar(_identifier_) is *false*, throw a *RangeError* exception.
+          1. Return ! CreateTemporalCalendar(_identifier_).
```

Comment:
Would this algorithm be more readable (and better communicate intent) if the return logic were combined?
```
        1. Let _inputType_ be Type(_temporalCalendarLike_).
        1. If _inputType_ is Object, then
          1. If _temporalCalendarLike_ has an [[InitializedTemporalDate]], [[InitializedTemporalDateTime]], [[InitializedTemporalMonthDay]], [[InitializedTemporalTime]], [[InitializedTemporalYearMonth]], or [[InitializedTemporalZonedDateTime]] internal slot, then
            1. Return _temporalCalendarLike_.[[Calendar]].
          1. If ? HasProperty(_temporalCalendarLike_, *"calendar"*) is *false*, return _temporalCalendarLike_.
          1. Set _temporalCalendarLike_ to ? Get(_temporalCalendarLike_, *"calendar"*).
          1. If Type(_temporalCalendarLike_) is Object and ? HasProperty(_temporalCalendarLike_, *"calendar"*) is *false*, return _temporalCalendarLike_.
        1. Let _identifier_ be ? ToString(_temporalCalendarLike_).
        1. If IsBuiltinCalendar(_identifier_) is *false*, then
          1. NOTE: If the original input is not an Object, accept a calendar annotation in its string representation.
          1. If _inputType_ is not Object, set _identifier_ to ? ParseTemporalCalendarString(_identifier_).
          1. If IsBuiltinCalendar(_identifier_) is *false*, throw a *RangeError* exception.
        1. Return ! CreateTemporalCalendar(_identifier_).
```